axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java

```diff
@@ -323,4 +328,30 @@ public void setParentAccountDomain(ActionRequest request, ActionResponse respons
       TraceBackService.trace(response, e);
     }
   }
+
+  public void viewMoveTemplates(ActionRequest request, ActionResponse response) {
+    Account account = request.getContext().asType(Account.class);
+
+    Query<MoveTemplateLine> moveTemplateLineList =
+        Beans.get(MoveTemplateLineRepository.class).findByAccount(account);
+    List<Long> moveTemplateIdList = List.of(0L);
+
+    if (moveTemplateLineList.count() > 0) {
+      moveTemplateIdList =
+          moveTemplateLineList.fetch().stream()
+              .map(MoveTemplateLine::getMoveTemplate)
+              .map(MoveTemplate::getId)
+              .distinct()
+              .collect(Collectors.toList());
+    }
```

Comment:
I think it's better here to fetch the moveTemplateLineList before counting it, this should allow to do one query instead of two